kombind/src/main/java/com/umairjavid/kombind/ui/KombindViewModel.kt

```diff
@@ -21,31 +21,7 @@ abstract class KombindViewModel(application: Application) : AndroidViewModel(app
         viewAction.value = viewActionQueue
     }
 
-    protected fun startActivity(intent: Intent) {
-        addViewAction(ViewAction(action = ViewAction.Action.START_ACTIVITY, intent = intent))
-    }
-
-    protected fun startActivityForResult(intent: Intent, requestCode: Int) {
-        addViewAction(ViewAction(action = ViewAction.Action.START_ACTIVITY_FOR_RESULT, intent = intent, requestCode = requestCode))
-    }
-
-    protected fun showDialog(builder: DialogFragmentBuilder<*>) {
-        addViewAction(ViewAction(action = ViewAction.Action.SHOW_DIALOG, builder = builder))
-    }
-
-    protected fun showDialog(builder: com.umairjavid.kombind.ui.v4.DialogFragmentBuilder<*>) {
-        addViewAction(ViewAction(action = ViewAction.Action.SHOW_DIALOG, v4builder = builder))
-    }
-
-    protected fun dismissDialog() {
-        addViewAction(ViewAction(action = ViewAction.Action.DISMISS_DIALOG))
-    }
-
-    protected fun dismissDialog(tag: String) {
-        addViewAction(ViewAction(action = ViewAction.Action.DISMISS_DIALOG_WITH_TAG, tag = tag))
-    }
-
-    protected fun finish() {
-        addViewAction(ViewAction(ViewAction.Action.FINISH))
+    fun action(viewAction: ViewAction) {
```

Comment:
can you actually revert this file and update each method? I like having the convenience methods in this class